### DISCUSSION TITLE
pdfjsLib not defined

### DISCUSSION THREAD

**Discussion by Gruski** [mozilla/pdf.js#17700] [Q&A]
I know this has been asked before in #14981 but solution there does not work. Here is the scenario:
```
<script src="Scripts2/pdf.mjs" type="module"></script>
<script type="module">
     // error occurs here pdfjsLib not defined       
     pdfjsLib.GlobalWorkerOptions.workerSrc = 'Scripts2/pdf.worker.mjs';
</script>
 ```
 Note that I am using "pdfjs-4.0.379-legacy-dist.zip" from releases
 Works on all desktop, all Android devices but not on iOS. I am testing on an iPhone 7 with v15.8 of iOS
 pdfjsLib not defined early in the pipeline as shown in code above.

**Comment by wangstar** [comment]
<script src="Scripts2/pdf.mjs" type="module" defer></script>
